ern-container-gen/src/generators/ios/hull/ElectrodeContainer/ElectrodeReactNative.m

```diff
@@ -190,7 +190,7 @@ - (void)startContainerWithConfiguration:(id<ElectrodePluginConfig>)reactContaine
 }
 
 - (void)loadCustomFonts {
-    NSMutableArray *fontPaths = [[NSBundle frameworkBundle] pathsForResourcesOfType:nil inDirectory:nil];
+    NSMutableArray *fontPaths = [[[NSBundle frameworkBundle] pathsForResourcesOfType:nil inDirectory:nil] mutableCopy];
```

Comment:
@clairewli 
Why do you want to `mutableCopy` here? Is the object not retained?
Doing `mutableCopy` increases the retain count and hence this is a memory leak..

Reply:
`pathsForResourcesOfType` returns an array that contains path names for all bundle resources (in this case). If nothing is matched it returns an empty array which is fine. 
So we shouldn't be doing `mutableCopy`.